store/store_test.go

```diff
@@ -33,7 +33,7 @@ import (
 type cleanupFunc func()
 
 // make a Commit with a single vote containing just the height and a timestamp
-func makeTestCommit(height int64, timestamp time.Time) *types.Commit {
+func makeTestCommit(block *types.Block, height int64, timestamp time.Time) *types.Commit {
```

Comment:
nit: seems a bit exessive to pass in the the whole block here.

Reply:
shoot, thought I responded to this. It is not necessary and has been removed https://github.com/lazyledger/lazyledger-core/pull/312/commits/3f1c2f2d5156373c02df815ca6c937b965378921

Reply:
no worries. I've noticed that you've updated it already.